cli/foundation-models/system/evaluation/fill-mask/fill-mask-pipeline.yml

```diff
@@ -29,6 +29,7 @@ jobs:
       input_column_names: input_string
       label_column_name: title
       device: gpu
```

Comment:
can you please check if we should set device to auto?

Reply:
can you also please check if we should be adding instance_type here?

Reply:
added device auto.
since it is a yaml file i can either run the job on compute or serverless not both, that's why havent added.
However i'll confirm

Reply:
thank you!!